server/liveblog/themes/themes_assets/default/js/theme/viewmodel.js

```diff
@@ -262,24 +310,21 @@ vm.init = function() {
   this.vm = getEmptyVm(settings.postsPerPage);
   this.vm.timeInitialized = new Date().toISOString();
 
-  function fetchLatestAndRender() {
-    vm.loadPosts({
-      fromDate: latestUpdate,
-      tags: selectedTags
-    })
-    .then(view.renderPosts)
-    .then(view.initGdprConsentAndRefreshAds);
-  }
-
   var isBlogOpen = LB.blog.blog_status === "open";
   var tenSeconds = 10 * 1000;
 
   if (isBlogOpen) {
```

Comment:
I think the sharing logic should work also if the blog is no longer open (archived).